api/tests/routes/pro/delete_stock_test.py

```diff
@@ -51,6 +88,24 @@ def test_delete_non_approved_offer_fails(self, app, db_session):
         assert response.json["global"] == ["Les offres refusées ou en attente de validation ne sont pas modifiables"]
 
 
+class Returns401Test:
+    def test_delete_stocks_not_connected(self, app, db_session):
+        nb_of_stocks = 1
+        offer = offers_factories.OfferFactory()
+        offerers_factories.UserOffererFactory(
+            user__email="pro@example.com",
+            offerer=offer.venue.managingOfferer,
+        )
+        batch_stocks = offers_factories.StockFactory.create_batch(nb_of_stocks, offer=offer)
```

Comment:
pas besoin de create_batch pour 1 élément

Reply:
Je pourrais faire un truc comme 
batch_stocks = [offers_factories.StockFactory(offer=offer)]
data = {"ids_to_delete": [stock.id for stock in batch_stocks]}

mais du coup ça perd en lisibilité, j'ai préféré mettre 3 stocks à chaque fois, après peut-être qu'on perd un peu de temps à chaque fois ?